backends/arm/scripts/install_models_for_test.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```

Comment:
What about updating this file? - https://github.com/pytorch/executorch/blob/de0554d86c7f2dc8724e85fab6c809cd7d7c7bb5/requirements-examples.txt

Also we have this :(
https://github.com/pytorch/executorch/blob/de0554d86c7f2dc8724e85fab6c809cd7d7c7bb5/examples/qualcomm/qaihub_scripts/stable_diffusion/install_requirements.sh#L2

Reply:
The TODO says it all. Thanks.